src/common/user-store/preferences-helpers.ts

```diff
@@ -194,6 +195,32 @@ const downloadBinariesPath: PreferenceDescription<string | undefined> = {
   },
 };
 
+const terminalFontSize: PreferenceDescription<number | undefined> = {
+  fromStore(val) {
+    return val;
```

Comment:
The benefit of having these descriptors is so that if the value is the default, we can return `undefined` (and thus not save them to disk). That way we can change the defaults in the future.